DigitalLearningSolutions.Web.Tests/Controllers/Login/LoginControllerTests.cs

```diff
@@ -497,6 +506,104 @@ public async Task Leading_trailing_whitespaces_in_username_are_ignored()
             A.CallTo(() => userService.GetUsersByUsername("test@example.com")).MustHaveHappened(1, Times.Exactly);
         }
 
+        [Test]
+        public async Task Admin_with_incorrect_password_increases_FailedLoginCount()
+        {
+            // Given
+            var adminUser = UserTestHelper.GetDefaultAdminUser();
+            A.CallTo(() => userService.GetUsersByUsername(A<string>._)).Returns((adminUser, new List<DelegateUser>()));
+            A.CallTo(() => loginService.VerifyUsers(A<string>._, A<AdminUser>._, A<List<DelegateUser>>._))
+                .Returns(new UserAccountSet());
+
+            // When
+            await controller.Index(LoginTestHelper.GetDefaultLoginViewModel());
+
+            // Then
+            A.CallTo(() => userDataService.UpdateAdminUserFailedLoginCount(7, 1)).MustHaveHappened();
+        }
+
+        [Test]
+        public async Task Admin_with_incorrect_password_increases_FailedLoginCount_and_returns_locked_view()

```

Comment:
Could this test be renamed to make it obvious that the locked view is only returned when the locked threshold is exceeded?